### ISSUE TITLE
Rework Home Page Logic

### ISSUE THREAD

**Issue by pivilartisant** [massalabs/station#889]
**Context**
The issue was created while trying to fix the home page install problem. Upon further investigation, we found that we had a bigger problem on our hands being : the current Home Page is not scalable and will certainly introduce bugs if we have multiple plugins due to the redundance of BE calls and lack of interoperability between pages. 

We have pushed a temporary simple solution to Fix the Home Page. This issue is a proposition for a rework of the Home Page logic : Hextra the dashboard & pluginWallet components from the ui-kit and set initialize them in the station repo than rework the current page to incorporate these changes. Nice to have : initialize a zustand store that will help us manage our different ressources.  

**How to**

- [ ] Extract components from ui-kit
- [ ] Initialize components in repo 
- [ ] Refactor the home page

Nice to have : 
- [ ] Set our ressources in zustand store to be accessible everywhere in our pages 

**QA testing**
Verify that we can install the plugin from the home page. Verify that the state of the plugin (installed or uninstalled) is the same on the home page and store page. Verify that

**Comment by Thykof** [comment]
I think we need to tackle this before adding/modifying feature to this page

**Comment by SlnPons** [comment]
Sorry @Thykof it doesn't answer my question. 😅
Let me rephrase: is it compliant with DoD? + release ready definition for the mainnet?

**Comment by Thykof** [comment]
from the DOD:
- You are proud of what will / has been merged
I believe this point is KO because we merge workaround and planned this refacto.
For the rest, everything is OK
